_build_cfg.yml

```diff
@@ -218,6 +218,15 @@ Topics:
         File: dynamically_provisioning_pvs
       - Name: Volume Security
         File: pod_security_context
+      - Name: Persistent Storage Troubleshooting
+        File: storage_troubleshooting
+  - Name: Persistent Storage Examples
+    Dir: storage_examples
+    Topics:
+      - Name: Overview
+        File: index
+      - Name: Sharing an NFS mount across two persistent volume claims
```

Comment:
Use title case:

Sharing an NFS Mount Across Two Persistent Volume Claims

However, I'd suggest a modified, shorter name here for the left-nav name, maybe:

Sharing an NFS Mount Across PVCs

But let the topic title in the shared_storage.adoc below be the full "Sharing an NFS Mount Across Two Persistent Volume Claims".